src/callbacks_stage/a_posteriori_limiter.jl

```diff
@@ -0,0 +1,53 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+
+"""
+    APosterioriLimiter()
+
+Perform antidiffusive stage for a posteriori IDP limiting.
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
+struct APosterioriLimiter end
+
+function (limiter!::APosterioriLimiter)(u_ode, integrator::Trixi.SimpleIntegratorSSP, stage)
+
+  limiter!(u_ode, integrator.p, integrator.t, integrator.dt, integrator.p.solver.volume_integral)
+end
+
+(::APosterioriLimiter)(u_ode, semi, t, dt, volume_integral::AbstractVolumeIntegral) = nothing
```

Comment:
This seems dangerous to me. If you pass this limiter, you want to it act, don't you?
```suggestion
```

Reply:
Yes, that's a good point. Thank you.